### PR TITLE
created cassandra db and seeded tables

### PR DESCRIPTION
**Pull request by benihime1458**

### CODE REVIEW

File: client/components/Map.jsx [hunk 1]
```diff
@@ -1,20 +1,20 @@
-import React from 'react';
-import CSSModules from 'react-css-modules';
-import styles from './css/map.css';
-import GoogleMapReact from 'google-map-react';
-import GOOGLE_MAP_KEY from '../config/GoogleMap.js';
+// import React from 'react';
+// import CSSModules from 'react-css-modules';
+// import styles from './css/map.css';
+// import GoogleMapReact from 'google-map-react';
+// import GOOGLE_MAP_KEY from '../config/GoogleMap.js';
```
Comment (serviolee) [inline client/components/Map.jsx:5]: just wondering, why are these lines commented out?

**Review by choiwa — APPROVED** [review]
looks good

File: client/components/Map.jsx (hunk 1 above)
Comment (benihime1458) [reply client/components/Map.jsx:5]: Not using google maps because its config and api key should be private and is included in gitignore.

Comment (serviolee) [reply client/components/Map.jsx:5]: ok, thanks for clarifying

**Pull request merged**